releng/platform.target

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<?pde version="3.8"?><target name="ELS Target" sequenceNumber="14">

```

Comment:
I don't mind the name change, however it could be used in the future by Tycho, once ls-api and gson 2.5 are properly published into a p2 repo.